adafruit_io/adafruit_io.py

```diff
@@ -565,6 +565,17 @@ def send_data(self, feed_key, data, metadata=None, precision=None):
         payload = self._create_data(data, metadata)
         self._post(path, payload)
 
+    def send_batch_data(self, feed, data_list):
+        """
+        Sends a batch array of data to a specified Adafruit IO feed
+        :param str feed_key: Adafruit IO feed key
+        :param list Data: Data list to send
```

Comment:
The kwarg on L568 is called `data_list`, this line should be `:param list data_list`.